modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -214,7 +214,19 @@ Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path, uint32_
 		}
 		return nullptr;
 	}
+
+#ifndef DISABLE_DEPRECATED
+	HashMap<StringName, Variant> opt = p_options;
+	if (!opt.has("animation/trimming")) {
+		opt.insert("animation/trimming", false);
+	}
+	if (!opt.has("animation/remove_immutable_tracks")) {
+		opt.insert("animation/remove_immutable_tracks", true);
+	}
+	return gltf->generate_scene(state, (float)opt["animation/fps"], (bool)opt["animation/trimming"], (bool)opt["animation/remove_immutable_tracks"]);
+#else
 	return gltf->generate_scene(state, (float)p_options["animation/fps"], (bool)p_options["animation/trimming"], (bool)p_options["animation/remove_immutable_tracks"]);
+#endif
```

Comment:
Seems fine overall, but isn't it a bit wasteful memory wise to copy the hashmap each time, when what we need is just two booleans?

It could be:
```suggestion
	bool trimming = p_options.has("animation/trimming") ? (bool)p_options["animation/trimming"] : false;
	bool remove_immutable = p_options.has("animation/remove_immutable_tracks") ? (bool)p_options["animation/remove_immutable_tracks"] : true;
	return gltf->generate_scene(state, (float)p_options["animation/fps"], trimming, remove_immutable);
```

Reply:
Good point...